tools/WebStack.xunit.targets

```diff
@@ -3,15 +3,13 @@
 
     <!-- This is a separate MSBuild file so that we can survive upgrades of the xunit NuGet package
          and also still work with NuGet Package Restore. -->
-    <ItemGroup>
-        <XunitMsBuildRunner Include="..\packages\**\xunit.runner.msbuild.dll"/>
-    </ItemGroup>
-    <UsingTask TaskName="Xunit.Runner.MSBuild.xunit" AssemblyFile="@(XunitMsBuildRunner)"/>
+    <Import Project="..\packages\**\xunit.runner.msbuild.props"/>
```

Comment:
xUnit made a breaking change in the name of their main assembly for this runner. Using the .props file avoids that change and should hopefully avoid breaking changes in the future (unless xUnit starts shipping multiple versions of this file 😈).